docs/components/tooltips.md

```diff
@@ -0,0 +1,131 @@
+<!--docs:
+title: "Tooltips"
+layout: detail
+section: components
+excerpt: "Tooltips display informative text when users hover over, focus on, or tap an element."
+iconId: slider
+path: /catalog/Tooltips/
+-->
+
+# Tooltips
+
+[Tooltips](https://material.io/components/tooltips) display informative text when users hover over, focus on, or tap an element.
+
+![Tooltip example in a text editor app](assets/tooltips/Tooltips_hero.png)
```

Comment:
alt-text needs updating (we're dropping the "example":

"Tooltip for "Bold" in a text editor menu"

Reply:
I added more descriptive labels for the images, PTAL